net-analyzer/monitoring-plugins/files/monitoring-plugins-mariadb-include-fix.patch

```diff
@@ -0,0 +1,24 @@
+--- a/plugins/check_mysql.c	2016-11-29 09:45:08.000000000 +0100
++++ b/plugins/check_mysql.c	2017-10-10 06:20:17.602585701 +0200
+@@ -42,6 +42,9 @@
+ #include "netutils.h"
+ 
+ #include <mysql.h>
++#if defined(MARIADB_BASE_VERSION) && defined(MARIADB_VERSION_ID)
++	#include <server/mysql_version.h>
```

Comment:
Please do not include this.  It is not meant to be used by clients ever.

Reply:
I know it's not a perfect solution, but the client/server header split happened in MariaDB after this release was done. Before the MariaDB header split we used MYSQL_PORT from a common include and now we use the very same MYSQL_PORT that is just defined elsewhere. 

Does it actually break anything? Does it change the behaviour?

Reply:
Yes, The headers in server are not used by the client library and likely to cause SIGSEV when you use them.

Reply:
As far as I understand, we only use that port definition. I've tested on a bunch of servers and check_mysql works properly, I don't understand why should it segfault.